src/api/index.tsx

```diff
@@ -204,7 +218,7 @@ export const getVersion = async ({
   addonId,
   versionId,
 }: GetVersionParams) => {
-  return callApi<ExternalVersionWithContent>({
+  return callApi<void, ExternalVersionWithContent>({
```

Comment:
This is a bit of a hassle. Is there a way to have it default to `void` instead of having to specify it? What if it's the second param instead of the first?

Reply:
I found out a better way! Thanks.